src/api/spec/policies/cloud/user/upload_job_policy_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe CommentPolicy do
+  let(:uploader) { create(:confirmed_user, login: 'burdenski') }
+  let(:admin_user) { create(:admin_user, login: 'admin') }
+  let(:staff_role) { Role.where(title: 'Staff').first_or_create(global: true) }
+  let(:staff_user) { create(:confirmed_user, login: 'staff', roles: [staff_role]) }
+  let(:user) { create(:confirmed_user, login: 'tom') }
```

Comment:
Nitpick, but I would drop setting the login here. It's not needed by the test and doesn't provide any additional information for readers of this test.